ic-os/components/BUILD.bazel

```diff
@@ -51,6 +51,7 @@ REPO_COMPONENTS = glob(
         "**/*.bzl",
         "**/*.md",
         "**/*.adoc",
+        "setupos-scripts/*",
```

Comment:
These scripts should not be excluded, was the new `BUILD.bazel` causing issues? It should have been captured by `**/BUILD.bazel`.

Reply:
Sure! What was the problem? I see the tests all passing with this change removed.

```diff
% git diff
diff --git a/ic-os/components/BUILD.bazel b/ic-os/components/BUILD.bazel
index 2e8d083e82..1e05b0f825 100644
--- a/ic-os/components/BUILD.bazel
+++ b/ic-os/components/BUILD.bazel
@@ -51,7 +51,6 @@ REPO_COMPONENTS = glob(
         "**/*.bzl",
         "**/*.md",
         "**/*.adoc",
-        "setupos-scripts/*",
     ],
 )
```

```
% bazel clean --expunge
INFO: Invocation ID: e6782ac9-c158-4203-909d-341953acd714
INFO: Starting clean (this may take a while). Consider using --async if the clean takes more than several minutes.

% bazel test ic-os/components/...
Starting local Bazel server and connecting to it...
INFO: Invocation ID: d4748570-d159-4ef7-848c-2396093eebd6

...

INFO: Analyzed 3 targets (206 packages loaded, 1727 targets configured).
INFO: Found 1 target and 2 test targets...
INFO: Elapsed time: 36.935s, Critical Path: 0.30s
INFO: 13 processes: 7 internal, 4 linux-sandbox, 2 local.
INFO: Build completed successfully, 13 total actions
//ic-os/components:check_unused_components_test                          PASSED in 0.0s
//ic-os/components/setupos-scripts:test-kernel-cmdline-function          PASSED in 0.1s

Executed 2 out of 2 tests: 2 tests pass.
There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones these are.
```

Reply:
OK.  removed the exclude.